lib/benches/benchmarks/aggregation.rs

```diff
@@ -14,6 +13,7 @@ use goodmetrics::{
         stream_sink::StreamSink,
     },
 };
+use tokio::sync::mpsc;
```

Comment:
If the std library is being thrown out, would you consider using flume? Flume tends to yield even better performance than tokio

Reply:
That would need a compelling benchmark - it's another dependency, and this library is probably already too wide with Tokio.